subprojects/docs/src/docs/userguide/more_about_tasks.adoc

```diff
@@ -450,6 +450,7 @@ Gradle supports this behavior out of the box through a feature it calls incremen
 
 How does incremental build work? And what does it take to make use of it in your own tasks? Let’s take a look.
 
+Note that changing the availability of an unavailable file (e.g. modifying the target of a broken symlink to a valid file, or vice versa), will be detected and handled by up-to-date check.
```

Comment:
I don't think this is a good place, it somewhat breaks the flow here. Maybe we could add another note under [How does it work](https://github.com/gradle/gradle/blob/b9a5ee61bd6f90a62c8d8046c9c2c40d7ff4c882/subprojects/docs/src/docs/userguide/more_about_tasks.adoc#how-does-it-work)?

Reply:
Moved